src/DGmethods/DGmodel.jl

```diff
@@ -8,124 +8,110 @@ end
 
 function (dg::DGModel)(dQdt, Q, param, t; increment=false)
   bl = dg.balancelaw
+  DFloat = eltype(Q)
   device = typeof(Q.Q) <: Array ? CPU() : CUDA()
 
   grid = dg.grid
-  topology = grid.topology
-
-  dim = dimensionality(grid)
-  N = polynomialorder(grid)
-  Nq = N + 1
-  Nqk = dim == 2 ? 1 : Nq
-  Nfp = Nq * Nqk
-  nrealelem = length(topology.realelems)
-
-  Qvisc = param.diff
-  auxstate = param.aux
+  Ω    = grid.topology.realelems
```

Comment:
I tend to think of `Ω` as the physical domain. Not sure that I have a better suggestion, just this is what comes to mind when I see this. Maybe `E` or `K` or something like this? Perhaps looking a standard finite element text could help (say Ciarlet?)

Reply:
Better to avoid Omega because we use it for planetary spin vector.

Reply:
I used Ω because this is what was used in the NDG book by Hesthaven and Warburton. Open to renaming it, just not sure what a good choice is

Reply:
I think either E or K should work. We tend to call it \Omega_e in DG notation but K also is used in the finite element formalism.

Reply:
In Hesthaven and Warburton (NDG): Ω is the domain (page 3), Ωh is the union of all the elements (page 19). The individual elements are denoted Dk (page 3). As far as I know, they do not have any notation for the set of elements {Dk}_{k=1}^{K} which is really what we are looking for here.

Ωh is close, but I tend to think of Ωh the domain covered by the set of elements and many different tessellations could have the same Ωh (especially since Ω ≃ Ωh, see page 19).

@fxgiraldo is Ωe usually a single element, as opposed to the collection/set of elements?

Just my two cents on the matter.

Reply:
I'll go with `E` since it is relatively intuitive. There is some overlap with energy, but hopefully we are never explicitly referring to the set of all elements and the energy explicitly in the same kernels.